Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
```

Comment:
Maybe call it "KeyBindingHandler" or something along those lines and
move other functionality and KeyBinding definitions into here?

Reply:
Why? which advantages would that provide?

Reply:
additionaly, it is not possible to fit any methods directly needed by any Unity Object (such as key binding buttons etc) here due to the static  properties here